### DISCUSSION TITLE
does HTTP2+Reality Work in xray 1.8.4 ?

### DISCUSSION THREAD

**Discussion by lambdaxyzt** [XTLS/Xray-core#2714] [Q&A] [answered]
does HTTP2+Reality Work in xray 1.8.4 ?

**Comment by chika0801** [comment]
https://github.com/chika0801/Xray-examples/tree/main/VLESS-H2-uTLS-REALITY

Do you check what's in the link, is it what you need?

**Reply by lambdaxyzt** [reply]
hi , yes , test it but v2rayN (version 6.21 , xray-core 1.8.4) say : 

`2023/11/11 13:59:41 [Warning] [595422450] app/proxyman/outbound: failed to process outbound traffic > proxy/vless/outbound: XTLS only supports TLS and REALITY directly for now.`

**Reply by chika0801** [reply]
You need to put the interface of how you fill in the node information in v2rayN. Take a screenshot and upload it to help you.

**Reply by lambdaxyzt** [reply]
![image](https://github.com/XTLS/Xray-core/assets/32255678/26afc471-3c32-440b-9828-1210b0944f98)

**Reply by chika0801** [reply, marked as answer]
`flow` This selection is blank.

**Reply by lambdaxyzt** [reply]
ohhhh , tnx very much

**Reply by ttzzt951** [reply]
xray has given up HTTP2 can not connect only singbox can work
